packages/preview/usaf-memo/0.0.1/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "usaf-memo"
```

Comment:
The chosen package name is too descriptive, in my opinion. Can you change it to make it more distinctive? See [the naming guidelines](https://github.com/typst/packages/blob/0488b38bb7fdfbce81981803ebbcf5612f8919f8/docs/manifest.md#naming-rules) if you need details about what is expected.